index.js

```diff
@@ -21,7 +21,7 @@ function querystring(query) {
   //
   for (;
     part = parser.exec(query);
-    result[decodeURIComponent(part[1])] = decodeURIComponent(part[2])
+    result[decodeURIComponent(part[1])] = part[2] ? decodeURIComponent(part[2]) : null
```

Comment:
Heh, I looked at a few others (like [query-string](https://www.npmjs.com/package/query-string)) which returned null.  Picked the wrong ones it seems 😄